apps/challenges/views.py

```diff
@@ -221,7 +221,7 @@ def get_all_challenges(request, challenge_time):
     Returns the list of all challenges
     """
     # make sure that a valid url is requested.
-    if challenge_time.lower() not in ("all", "future", "past", "present"):
+    if challenge_time.lower() not in ("all", "future", "past", "present", "featured"):
```

Comment:
It's a bad practice to add `featured` as one of the value of `challenge_time` attribute. Instead please use `featured` as a different query parameter in this endpoint.